### PR TITLE
libp2p updates:  1.2 goal

### PR DESCRIPTION
**Pull request by enoldev**

### CODE REVIEW

File: content/en/curriculum/libp2p/core-concepts/index.md [hunk 1]
```diff
@@ -1,97 +1,113 @@
...
+Consider the Internet router that you have at home. You probably have several devices connected to it (laptops, mobile phones, or even the fridge).
+When you join the Internet, you are given an address identifier, which is called IP.
+However, the number of IPs in the Internet is limited, so every device does not get a public IP address.
+Instead, your router creates a private network in your home and assigns a private IP to every device. Then, the router is responsible for managing the incoming and outgoing connection from the Internet.
+This is called **NAT (Network Address Traversal)**. For example, consider the following diagram:
 
-Circuit relay is a transport protocol that routes traffic between two peers over a third-party “relay” peer, when NAT Traversal and hole punching aren't an option, [the circuit relay can be used to connect them](https://blog.aira.life/understanding-ipfs-circuit-relay-ccc7d2a39).
+![NAT example](nat.png)
 
+The router is connected to the Internet, and is assigned a public IP address. Three devices are connected to the route, which manages the incoming and outgoing data.
```
Comment (mrodriguez3313) [inline content/en/curriculum/libp2p/core-concepts/index.md:53]:
```suggestion
The router is connected to the Internet, and is assigned a public IP address. Three devices are connected to the router, which manages the incoming and outgoing data.
```

File: content/en/curriculum/libp2p/core-concepts/index.md [hunk 2]
```diff
@@ -1,97 +1,113 @@
...
+You can read more in the [libp2p docs](https://docs.libp2p.io/concepts/peer-id/) or later in the Launchpad curriculum.
 
-#### [Security](https://docs.libp2p.io/concepts/security-considerations/)
+### Addressing
 
-libp2p makes it simple to establish [encrypted, authenticated communication channels](https://github.com/protocol/launchpad/blob/main/docs/secure-comms/README.md) between two peers, but there are other important security issues to consider when building robust peer-to-peer systems.
+In order to establish connections with a peer, it is necessary to know its location.
+For example, when you access `https://127.0.0.1:8080/my-image.png`, you are requesting access to the `127.0.0.1` IP address, at the `8080` port by using the HTTPs protocol.
 
+Although this is a valid system, you have to infer some data: what protocol is behind `127.0.0.1`? or what protocol is behind `8080`?
```
Comment (mrodriguez3313) [inline content/en/curriculum/libp2p/core-concepts/index.md:83]:
```suggestion
Although this is a valid system, you have to infer some data about what protocol is behind `127.0.0.1`? And what protocol is behind `8080`?
```

File: content/en/curriculum/libp2p/core-concepts/index.md [hunk 3]
```diff
@@ -1,97 +1,113 @@
...
 
-**[Multiplex Implementations](https://docs.libp2p.io/concepts/stream-multiplexing/#implementations)**
+### Publish/Subscribe
 
-Implementations of the multiplexing module include mplex protocol developer for libp2p, yamux by Hashicorp, quic transport protocol which includes a multiplexer, SPDY by Google, and more.
+libp2p also offers publish/subscribe capabilities by using the _topic_ abstraction.
+You can consider that a _topic_ is a bucket where messages for a specific use case are sent.
+For example, in a chat, you can create a "animals" topic, where only messages related to animals are sent.
 
+To receive messages, peers subscribe to topics they are interested in. Peer only receive messages from topics they are subscribed to.
```
Comment (mrodriguez3313) [inline content/en/curriculum/libp2p/core-concepts/index.md:109]:
```suggestion
To receive messages, peers subscribe to topics they are interested in. Peer only receive messages from topics they are subscribed to.
```
```suggestion
To receive messages, peers subscribe to topics they are interested in.
```

**Review by mrodriguez3313 — CHANGES_REQUESTED** [review]
10 minor spelling and sentence changes.

**Comment by enoldev** [comment]
@mrodriguez3313 About the capitalization of "libp2p", I was told it should be lowercase (probably marketing and brand), but you can bring it up in the content stand-up to see what's our standard

**Comment by enoldev** [comment]
Merging to use this content in upcoming libp2p updates PRs

**Pull request merged**